tests/php/modules/contact-form/test-class.grunion-contact-form.php

```diff
@@ -38,7 +38,7 @@ public function setUp() {
 
 		$post_id = $this->factory->post->create( array(
 			'post_status' => 'draft',
-			'post_author' => strval( $author_id )
+			'post_author' => (string) $author_id
```

Comment:
This will cause issues with our linter:

```suggestion
			'post_author' => (string) $author_id,
```